tsl/test/sql/dist_ddl.sql

```diff
@@ -150,6 +145,40 @@ INSERT INTO disttable VALUES
 
 SELECT * FROM show_chunks('disttable');
 
+-- Rename column
+ALTER TABLE disttable RENAME COLUMN description TO descr;
+SELECT * FROM test.show_columns('disttable')
+WHERE "Column"='descr';
+
+SELECT * FROM test.remote_exec('{ data_node_1 }', $$
+	   SELECT chunk.relid AS chunk_relid,
+	   		  (SELECT "Column" AS col FROM test.show_columns(chunk.relid) WHERE "Column"='descr')
+	   FROM (SELECT "Child" AS relid FROM test.show_subtables('disttable') LIMIT 1) chunk
+$$);
+
+-- Rename constraint
+ALTER TABLE disttable ADD CONSTRAINT device_check CHECK (device > 0);
+ALTER TABLE disttable RENAME CONSTRAINT device_check TO device_chk;
+SELECT * FROM test.show_constraints('disttable')
+WHERE "Constraint"='device_chk';
+
+SELECT * FROM test.remote_exec('{ data_node_1 }', $$
+       SELECT chunk.relid AS chunk_relid,
+	   		  (SELECT "Constraint" AS constr FROM test.show_constraints(chunk.relid) WHERE "Constraint"='device_chk')
+	   FROM (SELECT "Child" AS relid FROM test.show_subtables('disttable') LIMIT 1) chunk
+$$);
+
+-- Rename index
+ALTER INDEX disttable_description_idx RENAME to disttable_descr_idx;
+SELECT * FROM test.show_indexes('disttable')
+WHERE "Index"='disttable_descr_idx'::regclass;
+
+SELECT * FROM test.remote_exec('{ data_node_1 }', $$
+	   SELECT chunk.relid AS chunk_relid, (test.show_indexes(chunk.relid)).*
+	   FROM (SELECT "Child" AS relid FROM test.show_subtables('disttable') LIMIT 1) chunk
+$$);
```

Comment:
Same here. I do follow if this was a missing test, new functionality that you test, or anything else. It would be good if you could clarify this in the commit message.